front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/ProjectCarrousel/LightProjectCard.tsx

```diff
@@ -115,34 +114,37 @@ const LightProjectCard = ({ project, ml, mr, onKeyDown }: Props) => {
           )}
         </ProjectImageContainer>
       </Box>
-      <Title variant="h4" mt="8px" mb="0px">
-        {title}
+      <Title variant="h4" as="h3" mt="8px" mb="0px">
+        {truncate(title, 50)}
       </Title>
-      <Box mt="8px">
-        <Icon
-          name="clock-circle"
-          fill={colors.blue500}
-          height="16px"
-          mr="4px"
-          ml="-4px"
-          transform="translate(0,-1)"
-        />
-        <Text color="blue500" fontWeight="bold" display="inline" m="0">
-          {end_at ? (
-            <PhaseTimeLeft currentPhaseEndsAt={end_at} />
-          ) : (
-            <>{formatMessage(messages.noEndDate)}</>
-          )}
-        </Text>
-        <Text mt="4px" mb="0px" color="textSecondary" fontWeight="bold">
-          {getCTAMessage({
-            phase: phase.data,
-            actionDescriptors: project.attributes.action_descriptors,
-            localize,
-            formatMessage,
-          })}
-        </Text>
-      </Box>
+      {phase && (
+        <Box mt="8px">
+          <Box display="flex" flexDirection="row" alignItems="center">
+            <Icon
+              name="clock-circle"
+              fill={colors.blue500}
+              height="16px"
+              mr="4px"
+              ml="-4px"
+            />
```

Comment:
Just want to double-check that we need this negative margin? Is there a way to not use it?

Reply:
Not really, this margin is added by our `Icon` component for some reason. Sometimes you just need to tweak icons a bit in terms of positioning.